### PR TITLE
Add string.h to remove compile error

### PR DESCRIPTION
**Pull request by joel16**
memset and memcpy requires string.h

### CODE REVIEW

File: nx/source/services/set.c [hunk 1]
```diff
@@ -5,6 +5,8 @@
  * @author yellows8
  * @copyright libnx Authors
  */
+#include <string.h>
+
```
Comment (yellows8) [inline nx/source/services/set.c]: remove newline

Comment (joel16) [reply nx/source/services/set.c]: I was just following the way it's used in acc.c -> https://github.com/switchbrew/libnx/blob/master/nx/source/services/acc.c#L1 but okay.

Comment (yellows8) [reply nx/source/services/set.c]: Heh, no newline elsewhere :|

Comment (yellows8) [reply nx/source/services/set.c]: Er nvm revert this, original didn't have newline & nothing(?) else has newline here either.

**Pull request merged**